### ISSUE TITLE
Ranking de boston

### ISSUE THREAD

**Issue by dmonter** [pakopa/p4k0bot_requests#2]
Eso

**Comment by pakopa** [comment]
Interesante, evaluaré las posibles soluciones.

**Comment by dmonter** [comment]
pako espabila

**Comment by treezio** [comment]
lmao

**Comment by dmonter** [comment]
pako cabron dale caña a esto q el @treezio lo necesita

**Comment by treezio** [comment]
@pakopa tenemos ETA para esto ?

**Comment by pakopa** [comment]
No, pero describid que teneis en mente por si un dia me pongo y lo implemento.

Hay una version beta del pakobot con mejoras pero bastante limitado de funcionalidad @cigalabot

**Comment by treezio** [comment]
Sistema de gambleo para apostar ganador de precio y frescura del calamar boston entre mercamadrid y mercabarna.

query mercabarna 

![Image](https://github.com/user-attachments/assets/a5a6205b-6341-4850-b495-c36ff1cdb286)

**Comment by treezio** [comment]
no se si @dmonter da el approval

**Comment by treezio** [comment]
> No, pero describid que teneis en mente por si un dia me pongo y lo implemento.
> 
> Hay una version beta del pakobot con mejoras pero bastante limitado de funcionalidad @cigalabot

@pakopa me comentan de producto que @cigalabot no funciona, aviso al equipo de on-call

**Comment by dmonter** [comment]
> no se si [@dmonter](https://github.com/dmonter) da el approval

palante, con esta feature nos ponemos top del mercado del marisco mundial

**Comment by treezio** [comment]
perfecto, cuando tengamos un MVP definimos los KPIs para los quick wins prioritarios. Gracias a ambos

**Comment by treezio** [comment]
@pakopa tenemos fecha estimada ?
